packs/feats/death-roll.json

```diff
@@ -31,7 +31,25 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "attack-roll",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Werecreature.Werecrocodile.DeathRoll.ToggleLabel",
+                "option": "death-roll",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "death-roll",
```

Comment:
Didn't predicate off environment, as it might have been confusing why the toggle wasn't doing anything if the environment wasn't set.